subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,22 +34,34 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 <a name="cache"></a>
 ## Gradle's Dependency Cache can be relocated
 
-The Gradle Dependency cache under `$GRADLE_HOME/caches/modules-2` can now be relocated to another directory or host for dependencies cached by Gradle 6.1 and later. When moved to a new location or seeded into a host image, builds using the dependency cache will not need to access the network to download artifacts or metadata if the dependencies have already been downloaded. 
+It has become a common practice to run builds in a CI/CD pipeline with ephemeral agents, meaning single use environments.
```

Comment:
```suggestion
It has become a common practice to run builds in a CI/CD pipeline with ephemeral agents.
```
I think this misses a verb - something like ", that is using single use environments" or ", that is running builds on single use enviornments". Or put it into the next sentence:

This practice of running builds in a _single use environment_ is used to improve reliability and reduce potential interactions between builds.